force-di/main/classes/di_Binding.cls

```diff
@@ -67,12 +66,7 @@ public abstract class di_Binding implements Comparable {
      * Gets the instance of the thing the binding points to
      **/
     public Object getInstance(Object params) {
-        if(IsProvider) {
-            return newInstance(params);
-        } else if(Injected==null) {
-            Injected = newInstance(params);
-        }
-        return Injected;
+        return newInstance(params);
```

Comment:
**NOT backward compatible**

If we have a framework user who's relying upon this cached functionality, then this change will break their app.

I suggest overloading the 'getInstance' by adding a Boolean parameter with a name like "useCache".  "useCache" equal to True executes the current functionality.  "useCache" equal to False returns a unique instance.  With such, current users are not affected, whereas those explicitly wanting a no-cache implementation -- Chip -- are enabled too.

@ctchipps @ImJohnMDaniel

Reply:
@stohn777 Agreed, had this conversation with JD. I am working to update so that we preserve backwards compatibility. I have made some updates to the PR.